apps/dashboard/app/views/files/index.json.jbuilder

```diff
@@ -3,6 +3,7 @@ json.url files_path(@path).to_s
 
 #TODO: support array of shell urls, along with the default shell url which could be above
 json.shell_url OodAppkit.shell.url(path: @path.to_s).to_s
+json.shell_dropdown_html render partial: 'shell.html.erb', locals: { cluster: OodAppkit.clusters }
```

Comment:
You use the constant in the partial, so no need to pass the local "clusters" which is unused.